index.html

```diff
@@ -86,50 +125,246 @@
+                  .orient("bottom");
 
-    d3.select('#year-range').on('change', function () {
-      options.yearRange = d3.event.target.value;
-      update();
-    });
+    chart.svg.append("g")
+      .append("text")
+        .attr("x", width/2)
+        .attr("y", height+margin.top)
+        .attr("dy", ".71em")
+        .style("text-anchor", "middle")
+        .text("Percent of Workers that are Women");
+
+      var gx = chart.svg.append("g")
+          .attr("class", "x axis")
+          .call(xAxis);
+
+      gx.selectAll("g").filter(function(d) { return d; })
+          .classed("minor", true);
+
+
+    var formatAsDollars = d3.format("$1,000")
+
+    var yAxis = d3.svg.axis()
+                      .scale(chart.y)
+                      .tickFormat(formatAsDollars)
+                      .tickSize(width)
+                      .orient("right")
+                      .ticks(4);
+
+    chart.svg.append("g")
+      .append("text")
+        .attr("transform", "rotate(-90)")
+        .attr("x", -1*(height/2))
+        .attr("y", -1*(width/11))
+        .attr("dy", ".71em")
+        .style("text-anchor", "middle")
+        .text("Median Weekly Earnings");
+
+      var gy = chart.svg.append("g")
+          .attr("class", "y axis")
+          .call(yAxis);
+
+      gy.selectAll("g").filter(function(d) { return d; })
+          .classed("minor", true);
+
+      gy.selectAll("text")
+          .attr("x", -35);
+
+      chart.tooltip2 = d3.tip()
+        .attr('class', 'tooltip')
+        .html(function (d) {return d.SOCName; });
+//        .html(function (d) {return d.SOCName + "<br> Earn " + d.MedEarn; });
+
+      chart.svg.call(chart.tooltip2);
+    
+      chart.tooltip = d3.tip()
+        .attr('class', 'tooltip')
+        .offset([-6,0])
+        .html(function (d) {return d.length + ' Occupations' ; });
+
+      chart.svg.call(chart.tooltip);
+
+     chart.svg.append("g")
+        .append("text")
+        .attr("x", width+5)
+        .attr("y", chart.y(870))
+        .attr("dy", ".71em")
+        .style("text-anchor", "start")
+        .text("Total Median");
+
+     chart.svg.append("g")
+        .append("text")
+        .attr("x", width+5)
+        .attr("y", chart.y(770))
+        .attr("dy", ".71em")
+        .style("text-anchor", "start")
+        .text("Earnings"); 
+
+      chart.update()
   }
 
-  function update() {
+  Chart.prototype.update = function () {
+    var chart = this;
+    var t = d3.transition().duration(TRANSITION_DURATION)
+
     // TRANSFORM
     txData = data.slice();
 
-    if (options.yearRange !== 'all') {
-      var yearRange = options.yearRange.split('-');
+    if (options.SOCGroup !== 'all') {
+      var SOCGroup=options.SOCGroup;
       txData = txData.filter(function (d) {
-        return d.year >= yearRange[0] && d.year <= yearRange[1];
+        return d.SuperGroup === SOCGroup;
       });
     }
 
+
+
     // SCATTERPLOT
 
+    var points=chart.svg.selectAll('.point')
+      .data(txData)
+
```

Comment:
So there's a really minor issue with your transitions here. Note that by default, every time the update function runs, it's transitioning the circles using the index. So the circle corresponding to the first row of data in the prior subset is becoming the circle corresponding to the first row of the new subset.

This doesn't really make that much sense for this data, since these transitions generally indicate connectedness (one thing moving between various conditions).

You might instead want:
```js
.data(txData, function(d) { return d.unique_key } )
```

Where unique_key was a column of data that's unique for each row. 

A transition that makes more sense might be having the new dots expand from radius of 0 to their full radius, while the old dots fade out (diminishing radius or losing opacity, or something).